src/resolvers/FlyerResolver.ts

```diff
@@ -202,12 +203,16 @@ class FlyerResolver {
       startDate,
       title,
     );
+    NotificationRepo.notifyFlyersForOrganizations(flyer.id, ' just added a new flyer!');
+    return flyer;
   }
 
   @Mutation((_returns) => Flyer, {
     description: `Delete a flyer with the id <id>.`,
   })
   async deleteFlyer(@Arg('id') id: string) {
+    const flyer = await FlyerRepo.getFlyerByID(id);
```

Comment:
I believe `FlyerRepo.deleteFlyer` returns the deleted Flyer so you can just do
```
const flyer = await FlyerRepo.deleteFlyer(id);
NotificationRepo.notifyFlyersForOrganizations(flyer.id, ' just deleted a flyer');
return flyer;
```

Reply:
I think FlyerRepo.deleteFlyer returns an empty flyer because calling notifyFlyersForOrganization on the result from FlyerRepo.deleteFlyer(id); returns a null exception